### PR TITLE
Fixes tabs dropdown button now shown onResize

### PR DESCRIPTION
**Pull request by AnshulMalik**
Associated Issue: #2004 

### Summary of Changes

* Added onResize with splitter

### Test Plan

- [x] After resize, dropdown button appears

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -88,6 +108,7 @@ class App extends Component {
         minSize: 10,
         maxSize: "50%",
         splitterSize: 1,
+        onResize: onResize.bind(this, "start"),
```
Comment (jasonLaster) [inline src/components/App.js]: hmm, maybe we should change this to be: 

```js
onResize: size => this.setState({startPanelSize: size})
```

it's a little clenaer

File: src/components/App.js [hunk 2]
```diff
@@ -27,15 +27,23 @@ const WelcomeBox = createFactory(require("./WelcomeBox").default);
...
   renderEditorPane: Function;
   renderVerticalLayout: Function;
 
   constructor(props) {
     super(props);
-    this.state = { horizontal: verticalLayoutBreakpoint.matches };
+    this.state = {
+      horizontal: verticalLayoutBreakpoint.matches,
+      startPanelSize: 0,
+      endPanelSize: 0,
```
Comment (jasonLaster) [inline src/components/App.js]: I think @wldcordeiro might recommend that we put this state in the UI state. 

What do you think?

File: src/components/App.js [hunk 3]
```diff
@@ -61,12 +69,17 @@ class App extends Component {
...
     return dom.div(
       { className: "editor-pane" },
       dom.div(
         { className: "editor-container" },
-        EditorTabs({ startPanelCollapsed, endPanelCollapsed, horizontal }),
+        EditorTabs({
+          startPanelCollapsed,
+          endPanelCollapsed,
+          horizontal,
+          endPanelSize,
```
Comment (jasonLaster) [inline src/components/App.js]: is `endPanelSize` used?

**Comment by AnshulMalik** [comment]
I'll create a PR in devtools core, then it'll work fine.

File: src/components/App.js (hunk 3 above)
Comment (wldcordeiro) [inline src/components/App.js]: Yeah. 😛  https://github.com/devtools-html/debugger.html/pull/2492/files#diff-57aa11691094c3295e1f9dcbe8432ba2R113

File: src/components/App.js (hunk 2 above)
Comment (wldcordeiro) [inline src/components/App.js]: I wouldn't be against that, we already store that collapse state there.

File: src/components/App.js (hunk 3 above)
Comment (AnshulMalik) [inline src/components/App.js]: It's working with updated devtools-core splitter package :)

**Comment by AnshulMalik** [comment]
Thanks for the tweaks :)

**Pull request merged**